### PR TITLE
Release version 0.2.0

### PR DESCRIPTION
**Pull request by marimeireles**

### CODE REVIEW

**Comment by marimeireles** [comment]
@SylvainCorlay just published the [pypi](https://pypi.org/project/ipycytoscape/) and [npm](https://www.npmjs.com/package/jupyter-cytoscape) does it LGTM for you?

**Comment by marimeireles** [comment]
Maybe @martinRenou can check if my flow is correct?

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "jupyter-cytoscape",
-  "version": "0.1.5",
-  "description": "A Cytoscape Jupyter Widget",
+  "version": "0.2.0",
+  "description": "A Custom Jupyter Widget Library",
```
Comment (SylvainCorlay) [inline package.json:4]: I think mentioning cytoscape was more descriptive here.

**Comment by SylvainCorlay** [comment]
This looks good to me. I left a minor comment.

**Comment by marimeireles** [comment]
Thanks, I didn't mean to change that, actually.
It was a mistake
Thanks Sylvain! =)

**Pull request merged**